static-web-apps/tutorial.md

```diff
@@ -141,67 +136,66 @@ title: Create an Azure Static Web App
+In Azure, a Resource Group is a logical structure that holds resources usually related to an application or a project. A Resource Group can contain virtual machines, storage accounts, web applications, databases and more.
 </div>
 
-* Give a name to your Static Web App.
-* Select the free plan (we won't need any feature in the standard plan for this workshop).
+* Give a Name to your Static Web App.
+* Select the Free Plan (we won't need any feature in the Standard Plan for this workshop).
 * Select `West Europe` for your backend.
 
 <div class="box tip">
-It is recommended to host your backend in the region closests to your users.
+It is recommended to host your backend in a Region closest to your users.
 </div>
 
 * Select `GitHub` as a deployment source.
 
-We are going to host our website on GitHub. Later in the workshop, we will see how Static Web Apps will automaticaly deploy our website every time we push new code to our repository.
+We are going to host our website source code on GitHub. Later in the workshop, we will see how Static Web Apps will automaticaly deploy our website every time we push new code to our repository.
 
 * Sign in with your GitHub account and select the repository and the branch of your project.
 
 As we mentioned at the begining of the workshop, our app will have a backend and a frontend. In order for Static Web App to know what to deploy and where, we need to tell it where our apps are located in our repository.
 
-Azure Static Web Apps can handle several well-known frontend frameworks and therefore, "compile" your Angular, React or Hugo application before deploying it.
+Azure Static Web Apps can handle several well-known frontend frameworks and can "compile" your Angular, React or Hugo application before deploying them.
 
-In our case, we have a very simple Vanilla JavaScript appliation which does not require anything to run. So, let's choose `Custom`.
+In our case, we have a very simple JavaScript appliation which does not require anything to run. So, let's choose `Custom`.
 * In the `App location`, enter the `www/` folder as this is where our frontend is.
 * In the `Api location`, enter the `api/` folder as this is where our backend is.
 * In the `Output`, enter the `www/` folder as your frontend does not need any build system to run.
 
-![Enter GitHub information when creating SWA](media/swa-github.png)
+![Enter GitHub information when creating SWA](images/swa-github.png)
 
 * Click on `Review + Create` and then on `Create`.
 
-After a few minutes, your static web app will be created on Azure and your website deployed.
+After a few minutes, your Static Web App will be created on Azure and your website deployed.
 
-Once the resource is created, `pull` your code as a few files have been added to your GitHub repo by Azure.
+Once the Azure Resources are created, you should `pull` your repository in VSCode as a few files have been added to your GitHub repo by Azure.

```

Comment:
It only creates one resource, the Static Web App one

Reply:
OK, update the first component to "Once the resource is created..."